src/util/exit_codes.h

```diff
@@ -40,7 +40,6 @@ Author: Martin Brain, martin.brain@diffblue.com
 /// An (unanticipated) exception was thrown during computation.
 #define CPROVER_EXIT_EXCEPTION 6
 // should contemplate EX_SOFTWARE from sysexits.h
-#define CPROVER_EXIT_EXCEPTION_GOTO_INSTRUMENT 11
```

Comment:
This will lead to a change in exit code for `goto-instrument`. That doesn't bother me personally, but it's a change in behaviour, not sure if it needs to documented somewhere. Also, the comment above should be removed together with this `#define`

Reply:
1. It is documented in the commit message.

2. The comment is general to all `CPROVER_EXIT_EXCEPTION`, not specifically this one.